requirements.txt

```
@@ -4,3 +4,4 @@ paramiko
 pytest-testinfra
 pytest-durations
 pyyaml
+-e .
```

```suggestion
obsah >= 1.1
```

And it should be sorted